p2p/src/main/java/bisq/network/p2p/peers/PeerManager.java

```diff
@@ -410,29 +570,22 @@ private void removeSuperfluousSeedNodes() {
         }
     }
 
-
     ///////////////////////////////////////////////////////////////////////////////////////////
     // Reported peers
     ///////////////////////////////////////////////////////////////////////////////////////////
 
-    private boolean removeReportedPeer(Peer reportedPeer) {
-        boolean contained = reportedPeers.remove(reportedPeer);
+    private void removeReportedPeer(Peer reportedPeer) {
+        reportedPeers.remove(reportedPeer);
         printReportedPeers();
-        return contained;
     }
 
-    @SuppressWarnings("UnusedReturnValue")
-    @Nullable
-    private Peer removeReportedPeer(NodeAddress nodeAddress) {
+    private void removeReportedPeer(NodeAddress nodeAddress) {
         List<Peer> reportedPeersClone = new ArrayList<>(reportedPeers);
         Optional<Peer> reportedPeerOptional = reportedPeersClone.stream()
                 .filter(e -> e.getNodeAddress().equals(nodeAddress)).findAny();
         if (reportedPeerOptional.isPresent()) {
             Peer reportedPeer = reportedPeerOptional.get();
             removeReportedPeer(reportedPeer);
-            return reportedPeer;
-        } else {
-            return null;
         }
```

Comment:
Easier way
```
        reportedPeersClone.stream()
                .filter(e -> e.getNodeAddress().equals(nodeAddress))
                .findAny()
                .ifPresent(this::removeReportedPeer);
```

Reply:
Will apply. Just did not intend to refactor too much but I agree....